### PR TITLE
Adding Methyl Decanoate Data

### PR DESCRIPTION
**Pull request by bradcarr23**
3 data files from "A shock tube study of methyl decanoate autoignition at elevated pressures" from the Combustion and Flame Journal

### CODE REVIEW

**Comment by bryanwweber** [comment]
@kyleniemeyer Can you take a look at this? This is related to the idea we had for WSSCI, although this might be all or most of the data, so we might need to pick a different fuel...

File: methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_0.5.yaml [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+      ignition-delay:
+        - 4687 us
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 17.1 atm
+        - uncertainty-type: relative
+          uncertainty: 0.02
+      pressure-rise:
+        - 3 1/ms
```
Comment (kyleniemeyer) [inline methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_0.5.yaml]: should probably be `3.0 1/ms` to reflect significant digits given by the paper

File: methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_0.5.yaml [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+      ignition-delay:
+        - 4482 us
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 15.6 atm
+        - uncertainty-type: relative
+          uncertainty: 0.02
+      pressure-rise:
+        - 3 1/ms
```
Comment (kyleniemeyer) [inline methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_0.5.yaml]: same comment as above, `3.0 1/ms`

File: methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_0.5.yaml [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+    - temperature:
+        - 1024 kelvin
+        - uncertainty-type: relative
+          uncertainty: 0.015
+      ignition-delay:
+        - 1401 us
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 16 atm
```
Comment (kyleniemeyer) [inline methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_0.5.yaml]: `16.0 atm`

File: methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.0.yaml [hunk 4]
```diff
@@ -0,0 +1,578 @@
...
+    - temperature:
+        - 987 kelvin
+        - uncertainty-type: relative
+          uncertainty: 0.015
+      ignition-delay:
+        - 1979 us
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 14 atm
```
Comment (kyleniemeyer) [inline methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.0.yaml]: `14.0 atm`

File: methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.0.yaml [hunk 5]
```diff
@@ -0,0 +1,578 @@
...
+    - temperature:
+        - 1174 kelvin
+        - uncertainty-type: relative
+          uncertainty: 0.015
+      ignition-delay:
+        - 177 us
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 18 atm
```
Comment (kyleniemeyer) [inline methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.0.yaml]: `18.0 atm`

File: methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.0.yaml [hunk 6]
```diff
@@ -0,0 +1,578 @@
...
+    - temperature:
+        - 1285 kelvin
+        - uncertainty-type: relative
+          uncertainty: 0.015
+      ignition-delay:
+        - 55 us
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 16 atm
```
Comment (kyleniemeyer) [inline methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.0.yaml]: `16.0 atm`

File: methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.5.yaml [hunk 7]
```diff
@@ -0,0 +1,350 @@
...
+    - temperature:
+        - 705 kelvin
+        - uncertainty-type: relative
+          uncertainty: 0.015
+      ignition-delay:
+        - 2611 us
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 16 atm
```
Comment (kyleniemeyer) [inline methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.5.yaml]: `16.0 atm`

File: methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.5.yaml [hunk 8]
```diff
@@ -0,0 +1,350 @@
...
+      ignition-delay:
+        - 3002 us
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 13.1 atm
+        - uncertainty-type: relative
+          uncertainty: 0.02
+      pressure-rise:
+        - 3 1/ms
```
Comment (kyleniemeyer) [inline methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.5.yaml]: `3.0 1/ms`

File: methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.5.yaml [hunk 9]
```diff
@@ -0,0 +1,350 @@
...
+    - temperature:
+        - 1047 kelvin
+        - uncertainty-type: relative
+          uncertainty: 0.015
+      ignition-delay:
+        - 539 us
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 16 atm
```
Comment (kyleniemeyer) [inline methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.5.yaml]: `16.0 atm`

**Comment by kyleniemeyer** [comment]
@bradcarr23 overall looks good! I left a few comments where significant digits got dropped from the data given in the paper (i.e., `16.0` turned into `16`). This is likely not important, as we don't use this information now, but I'd rather try and retain that information in case it somehow becomes useful in the future.

**Comment by kyleniemeyer** [comment]
Also, I wanted to check on whether the `ignition-type` was appropriate; the paper gives this on how ignition was detected:
> The ignition delay time was defined as the time interval between shock arrival and reflection at the end wall, determined using the side wall pressure and incident shock velocity measurements, and the onset of ignition at the end wall, defined by extrapolating the maximum slope in OH* emission to the baseline.

You have the `ignition-type` set as:
```
    ignition-type: &ign
        target: OH*
        type: d/dt max
```
and I'm wondering if the `type` should instead be `d/dt max extrapolated`.

**Comment by bradcarr23** [comment]
Professor Niemeyer,
Sorry it has taken so long to get back to you, I have been studying for finals, and getting settled back home for break. I will fix the significant digits on the next pull request. I also believe that the ignition-type should be "d/dt max extrapolated" due to the extrapolation of the OH* emission. I will change that now and submit a new request. Thank you!

**Comment by bryanwweber** [comment]
@bradcarr23 You can make some new commits on this branch and push it up to your copy. That will automatically update the pull request (PR)

**Comment by kyleniemeyer** [comment]
@bradcarr23 sounds good, no rush!

**Comment by bradcarr23** [comment]
@kyleniemeyer @bryanwweber I have added the new files to the branch and pushed it up to my copy. Now it says that 6 files were changed, rather than just the 3 new ones. Is this correct?

**Comment by bryanwweber** [comment]
@bradcarr23 It looks like you added the second set of files into the root directory, rather than into the `methyl-decanoate` directory. I bet if you move the new files into the subdirectory and replace the existing files, then make another commit, it will work out

**Comment by kyleniemeyer** [comment]
@bradcarr23 ok, this looks good to accept now, thanks!

**Pull request merged**